MenuApplets/Clock/main.cpp

```diff
@@ -79,25 +80,45 @@ class ClockWidget final : public GUI::Widget {
         painter.draw_text(event.rect(), time_text, Gfx::Font::default_font(), Gfx::TextAlignment::Center, palette().window_text());
     }
 
+    virtual void mousedown_event(GUI::MouseEvent& event) override
+    {
+        if (event.button() != GUI::MouseButton::Left)
+            return;
+        // check if the "date" section was clicked
+        if (event.x() <= rect().top_left().x() + m_date_section_width) {
```

Comment:
This is a bit weird. Let's make the whole date+time clickable, not just the date.
That matches what other systems do anyway. :)

Reply:
I figured we might have a Clock app some day.
Let us change this then.